driver/csiplugin/controllerserver.go

```diff
@@ -1123,10 +1123,14 @@ func (cs *ScaleControllerServer) getSnapshotCreateTimestamp(conn connectors.Spec
 		return timestamp, err
 	}
 
-	const longForm = "2006-01-02 15:04:05,000" // This is the format in which REST API return creation timestamp
+	// REST API returns creation timestamp without timezone info
+	// So we parse it with current location and return in UTC as kubernetes
+	// displays in UTC
+	const longForm = "2006-01-02 15:04:05"
 	//nolint::staticcheck
-	t, _ := time.Parse(longForm, createTS)
-	timestamp.Seconds = t.Unix()
+	location := time.Now().Location()
```

Comment:
I am not sure hence asking 
What will happen in case of remote cluster where timezone is different that in local cluster ?

Reply:
Thats a good point, we would break again in that case. Ideally this should be fixed in REST API. I have opened a defect for it- 
[STG Defect 255479: [REST API] - Snapshots GET should return correct timezone info in "created" field](https://jazz07.rchland.ibm.com:21443/jazz/web/projects/GPFS#action=com.ibm.team.workitem.viewWorkItem&id=255479)